src/main/java/persistence/sql/ddl/column/Columns.java

```diff
@@ -0,0 +1,53 @@
+package persistence.sql.ddl.column;
+
+import jakarta.persistence.Transient;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Columns {
+
+    private final List<EntityColumn> columns;
+
+    private Columns(List<EntityColumn> columns) {
+        this.columns = columns;
+
+        checkHasIdAnnotation(columns);
```

Comment:
```suggestion
        checkHasIdAnnotation(columns);
        this.columns = columns;
```

메모리가 할당 되기 전에 유효성 검증을 하는 것이 더 낫지 않을까요?